Mathlib/NumberTheory/ModularForms/Basic.lean

```diff
@@ -128,6 +128,11 @@ protected def CuspForm.copy (f : CuspForm Γ k) (f' : ℍ → ℂ) (h : f' = ⇑
   holo' := h.symm ▸ f.holo'
   zero_at_infty' A := h.symm ▸ f.zero_at_infty' A
 
+def CuspForm.toModularForm (f : CuspForm Γ k) : ModularForm Γ k :=
```

Comment:
I dont thik we should need this since we have `instance (priority := 99) [FunLike F ℍ ℂ] [CuspFormClass F Γ k] : ModularFormClass F Γ k where` further down the file.

Reply:
I see, I never looked too closely at the `Class` things, but I should. I've also removed this then